### PR TITLE
Add Certus One as a trusted validator for testnet

### PR DESCRIPTION
**Pull request by mvines**

### CODE REVIEW

**Comment by mvines** [comment]
cc: @leoluk

File: docs/src/clusters.md [hunk 1]
```diff
@@ -68,10 +68,12 @@ solana config set --url https://testnet.solana.com
 
 ##### Example `solana-validator` command-line
 ```bash
+$ trusted_validators=(--trusted-validator 5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on)  # testnet.solana.com
+$ trusted_validators+=(--trusted-validator Ft5fbkqNa76vnsjYNwjDZUXoTWpP7VYm3mtsaQckQADN) # Certus One
 $ solana-validator \
     --identity ~/validator-keypair.json \
     --vote-account ~/vote-account-keypair.json \
-    --trusted-validator 5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on \
+    "${trusted_validators[@]}" \
```
Comment (leoluk) [inline docs/src/clusters.md:76]: I suspect that this might confuse some entry-level validators - why the array?

Comment (mvines) [reply docs/src/clusters.md:76]: I wanted to associate a name with each of the trusted validators and bash won't let me to:
```bash
blah \ # invalid comment.  💔 
  blah

Comment (mvines) [reply docs/src/clusters.md:76]: Maybe I'll just create a table instead

**Comment by solana-grimes** [comment]
:scream: New commits were pushed while the automerge label was present.

**Pull request merged**